### PR TITLE
docs: added Github Star count button

### PR DESCRIPTION
**Pull request by YashasviChaurasia**
Added GitHub Star count below Get Started button on the website,
solves Issue#1619

Signed-off-by: Yashasvi Chaurasia <yashasvi12977@gmail.com>

### CODE REVIEW

**Comment by mturoci** [comment]
Hi @YashasviChaurasia, if you want to work on some issue, please comment on it and wait for the confirmation of the Wave team member first. This one is already taken.

See [contribution guidelines](https://github.com/h2oai/wave/blob/master/.github/CONTRIBUTING.md) prior to contributing.

Moreover, your PR should be runnable without throwing any errors.

**Comment by YashasviChaurasia** [comment]
Ok, sorry for the inconvenience caused.
Will look into it from next time.
Could you please specify the error for my understanding. 
Thank you

**Comment by mturoci** [comment]
Np. The problem is that whenever you add a new npm package (in this case `react-github-btn`), you also need to commit the changed `package-lock.json` file so that whoever picks up your changes only needs to run `npm ci` and will get the exact dependencies as you did.

File: website/src/pages/index.js
Comment (mturoci) [inline website/src/pages/index.js:114]: Formatting:

* remove extra blank lines
* line is quite long, separate attributes with newlines

**Review by mturoci — CHANGES_REQUESTED** [review]
Thanks @YashasviChaurasia, in addition to missing `package-lock`, a bit of extra formatting is also needed.

**Review by mturoci — CHANGES_REQUESTED** [review]
* When adding a new npm dependency, you need to commit both `package.json` and `package-lock.json` so after running `npm ci` to do the initial setup, you run `npm i <your-dep>` which edits both of the mentioned files which need to be commited.
* Please attach screenshots/gifs to the PR description so that we can see if the button is properly aligned and works as expected.
* add `Closes #<issue-number` clause to link the PR with the GH issue and automatically close it after merging.

Thanks!

**Comment by mturoci** [comment]
Closing due to inactivity.

**Pull request closed without merging**
